paratex/extractor.py

```diff
@@ -18,8 +18,7 @@ class Session:
     @staticmethod
     def make_url(session_id: int):
         BASE_URL = (
-            "http://transparencia.alesc.sc.gov.br/"
-            "presenca_plenaria_detalhes.php"
+            "http://transparencia.alesc.sc.gov.br/" "presenca_plenaria_detalhes.php"
```

Comment:
```suggestion
            "http://transparencia.alesc.sc.gov.br/presenca_plenaria_detalhes.php"
```

This max line size on black sounds a bit big no? I preferred the 2 line break here